From 126190957aa60f6723c2b183728366e8ced0e3d6 Mon Sep 17 00:00:00 2001
From: Jason Montleon <jmontleo@redhat.com>
Date: Thu, 26 Dec 2024 15:24:04 -0500
Subject: [PATCH 1201/1206] Remove depends so PWM_PXA can be enabled

---
 drivers/pwm/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/pwm/Kconfig b/drivers/pwm/Kconfig
index f0e1eb4a2c8d..29ce848605ed 100644
--- a/drivers/pwm/Kconfig
+++ b/drivers/pwm/Kconfig
@@ -501,7 +501,6 @@ config PWM_PCA9685
 
 config PWM_PXA
 	tristate "PXA PWM support"
-	depends on ARCH_PXA || ARCH_MMP || COMPILE_TEST
 	depends on HAS_IOMEM
 	help
 	  Generic PWM framework driver for PXA.
-- 
2.47.0

