From 9786e5e73b62302fe973302b2a5ba371c8905664 Mon Sep 17 00:00:00 2001
From: Jason Montleon <jmontleo@redhat.com>
Date: Thu, 26 Dec 2024 15:26:26 -0500
Subject: [PATCH 1201/1202] fix includes for timestamp

---
 include/uapi/linux/videodev2.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/uapi/linux/videodev2.h b/include/uapi/linux/videodev2.h
index e7c4dce39007..6b225422d7f5 100644
--- a/include/uapi/linux/videodev2.h
+++ b/include/uapi/linux/videodev2.h
@@ -58,7 +58,7 @@
 #define _UAPI__LINUX_VIDEODEV2_H
 
 #ifndef __KERNEL__
-#include <sys/time.h>
+#include <linux/time.h>
 #endif
 #include <linux/compiler.h>
 #include <linux/ioctl.h>
-- 
2.47.0

