diff --git a/drivers/thermal/k1x-thermal.c b/drivers/thermal/k1x-thermal.c
index eaf5d345628f..24c744710deb 100644
--- a/drivers/thermal/k1x-thermal.c
+++ b/drivers/thermal/k1x-thermal.c
@@ -342,7 +342,7 @@ static const struct of_device_id of_k1x_thermal_match[] = {
 	{ /* end */ }
 };
 
-MODULE_DEVICE_TABLE(of, of_hisi_thermal_match);
+MODULE_DEVICE_TABLE(of, of_k1x_thermal_match);
 
 static struct platform_driver k1x_thermal_driver = {
 	.driver = {
